pkg/aws/sts.go

```diff
@@ -202,3 +204,73 @@ func BuildOperatorRoleCommands(prefix string, accountID string, awsClient Client
 	}
 	return commands
 }
+
+func (c *awsClient) FindAccRoleArnBundles(version string) (map[string]map[string]string, error) {
```

Comment:
Can we break this function into smaller functions?
To make it easier to follow the logic :)

Reply:
Sure, let me look into that